typescript/looker/test/methods.spec.ts

```diff
@@ -50,7 +50,7 @@ const root = fs.existsSync(dataFile) ? '' : '../../'
 const testData = yaml.safeLoad(fs.readFileSync(`${root}${dataFile}`, utf8))
 const localIni = `${root}looker.ini`
 const users: Partial<IUser>[] = testData['users']
-const queries: Partial<IQuery>[] = testData['queries']
+const queries: Partial<IQuery>[] = testData['queries_system_activity']
```

Comment:
Do all the test pass using this testData section instead of `queries`?

Reply:
If you use 'queries' it will look for "thelook" which not everyone has. If you use 'queries_system_activity' it will use internal looker data. Are there other places in the tests where testData is set?

Reply:
It's the methods tests in the various SDKs that use the data.yml or data.yml.json file. Seems like we should remove our reliance on `thelook` being present and use the system activity model instead, and replace the data in `queries` with `queries_system_activity` model and remove `queries_system_activity` from data.yml.